tests/integration/test_oidc_client.py

```diff
@@ -0,0 +1,47 @@
+from urllib.parse import quote
+
+from flask import url_for
+import pytest
+import requests
+
+
+class TestOIDCClient(object):
```

Comment:
Need to update class to start with When rather than Test as pytest.ini updated to BDD style